gala 6.3.2 breaks wingpanel build

Building wingpanel master with gala 6.3.2 on NixOS fails with:

```
FAILED: wingpanel-interface/libwingpanel-interface.so.p/Main.c wingpanel-interface/libwingpanel-interface.so.p/DBusServer.c wingpanel-interface/libwingpanel-interface.so.p/BackgroundManager.c wingpanel-interface/libwingpanel-interface.so.p/FocusManager.c wingpanel-interface/libwingpanel-interface.so.p/Utils.c wingpanel-interface/wingpanel-interface.h wingpanel-interface/wingpanel-interface.vapi 
valac -C --vapidir=/build/source/vapi --pkg mutter-clutter-10 --pkg mutter-cogl-pango-10 --pkg mutter-cogl-10 --pkg libmutter-10 --pkg granite --pkg gala --color=always --directory wingpanel-interface/libwingpanel-interface.so.p --basedir ../wingpanel-interface --library wingpanel-interface --header wingpanel-interface/wingpanel-interface.h --vapi ../wingpanel-interface.vapi --define HAS_MUTTER338 --define HAS_MUTTER40 --define HAS_MUTTER41 --define HAS_MUTTER42 ../wingpanel-interface/Main.vala ../wingpanel-interface/DBusServer.vala ../wingpanel-interface/BackgroundManager.vala ../wingpanel-interface/FocusManager.vala ../wingpanel-interface/Utils.vala
gala.vapi:66.21-66.25: error: The symbol `Posix' could not be found
   66 |                 public GLib.SList<Posix.pid_t?> get_pids ();
      |                                   ^~~~~                     
Compilation failed: 1 error(s), 0 warning(s)
[206/235] Compiling Vala source ../lib/Indicator.vala ../lib/IndicatorManager.vala ../lib/Widgets/Container.vala ../lib/Widgets/OverlayIcon.vala ../lib/Widgets/Separator.vala ../lib/Widgets/Switch.vala
../lib/IndicatorManager.vala:172.44-172.64: warning: `GLib.ModuleFlags.BIND_LAZY' has been deprecated since vala-0.46. Use LAZY
  172 |         Module module = Module.open (path, ModuleFlags.BIND_LAZY);
      |                                            ^~~~~~~~~~~~~~~~~~~~~  
Compilation succeeded - 1 warning(s)
ninja: build stopped: subcommand failed.
```

It looks like the problem is introduced in 7c540a2a0669fb7de84ed1071f85f5d331398040 and https://github.com/elementary/wingpanel/commit/8be50d17eb10458f89a6c5e029c81e6f3042c6f9 does not fix the issue for me, I will try look into it.

--- a/lib/gala.deps.in
+++ b/lib/gala.deps.in
@@ -1,4 +1,5 @@
 gdk-pixbuf-2.0
 glib-2.0
 gobject-2.0
+posix
 @MUTTER_DEP@

